tools/wltests/agendas/sched-evaluation-full-traced.yaml

```diff
@@ -0,0 +1,124 @@
+# This is an example agenda which shows a possible (rather large) set of
```

Comment:
In the commit message: "by deafult in the "standard" sched-evaluation-full's agenda" -> "by default in the "standard" sched-evaluation-full agenda"